### PR TITLE
For #284: var_siblings pattern implementation

### PR DESCRIPTION
**Pull request by paulodamaso**
For #284 
- implemented `var_siblings` pattern
- fixed test, `value()` method on pattern should return `List[LineNumber]` (see https://github.com/yegor256/aibolit/pull/239#discussion_r414660197)

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#346](https://github.com/yegor256/aibolit/issues/346) is assigned to @lyriccoder[/z](https://www.0crat.com/u/lyriccoder), here is [why](https://www.0crat.com/footprint/CT5UB7XAP/cd702587-0e38-4e5d-847c-6c46f7323113); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @acheshkov[/z](https://www.0crat.com/u/acheshkov) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: aibolit/patterns/var_siblings/var_siblings.py [hunk 1]
```diff
@@ -1,10 +1,20 @@
+import javalang
+import re
+from aibolit.utils.ast import AST
+
 
 class VarSiblings:
     def __init__(self):
         pass
```
Comment (lyriccoder) [inline aibolit/patterns/var_siblings/var_siblings.py:9]: 1) Remove P27 pattern from config.py
2) Check tests (see `CI_pipeline` log), they take too much time. Seems smth is wrong with your code.

**Comment by paulodamaso** [comment]
@lyriccoder Made the changes. I had to add a suppression for a pep8 check, please take a look at  https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

**Comment by paulodamaso** [comment]
@acheshkov I think that you could merge it?

**Comment by paulodamaso** [comment]
@acheshkov ping

**Comment by acheshkov** [comment]
@rultor merge

**Comment by paulodamaso** [comment]
@acheshkov please take a look at this one, it is already done

**Comment by paulodamaso** [comment]
@acheshkov Ping

**Comment by paulodamaso** [comment]
@acheshkov ping

**Comment by acheshkov** [comment]
@rultor merge

**Comment by 0crat** [comment]
Code review was too long (28 days), architects (@acheshkov) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Comment by 0crat** [comment]
Job audit: performer @lyriccoder[/z](https://www.0crat.com/u/lyriccoder)[/z](https://www.0crat.com/u/lyriccoder) didn't make CR comments

**Pull request merged**